server/proxy/enable.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/mattermost/mattermost-plugin-apps/server/utils/md"
 )
 
+// EnableApp is never called. Context vetting needs to be done by the caller via p.CleanUserCallContext, though we're temporarily doing it here instead.
```

Comment:
0/5 is the comment `EnableApp is never called` really helpful? It might be obsolete soon.

Reply:
I was thinking the same. Are you suggesting to remove the entire comment, or just that sentence?